R/learner_randomForestSRC_classif_imbalanced_rfsrc.R

```diff
@@ -16,7 +16,28 @@
 #' `r format_bib("obrien2019imbrfsrc", "chen2004imbrf")`
 #'
 #' @template seealso_learner
-#' @template example
+#' @examples
+#' # Define the Learner
+#' learner = mlr3::lrn("classif.imbalanced_rfsrc", importance = "TRUE")
```

Comment:
why TRUE in quotes?

Reply:
the param doesn't take the logical `TRUE`! See error:
```
Error in self$assert(xs, sanitize = TRUE) : 
  Assertion on 'xs' failed: importance: Must be element of set {'FALSE','TRUE','none','permute','random','anti'}, but types do not match (logical != character).
```